msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -229,10 +224,9 @@ The `NotificationSignal` API includes the following response codes:
 ## Meeting Details API
 
 > [!NOTE]
-> This feature is currently available in [public developer preview](../resources/dev-preview/developer-preview-intro.md) only.
+> Currently, the feature is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md) only.
 
-The Meeting Details API enables your app to get static meeting metadata. The metadata provides data points that don't change dynamically.
-The API is available through Bot Services.
+The Meeting Details API enables your app to get static meeting metadata. The metadata provides data points that don't change dynamically. The API is available through Bot Services.
```

Comment:
to get the static metadata about a meeting.

Reply:
Done